examples/cr-events.rs

```diff
@@ -93,9 +93,11 @@ fn main() {
                         new,
                         old: _,
                     } => (cr_type, new),
+                    _ => (CrType::Cr3, 0),
```

Comment:
This doesn't make sense.
If the event is not a Cr, we should do some error handling, and not convert it to fake CR3